src/views/DashboardView/DashboardView.js

```diff
@@ -123,9 +111,4 @@ const mapStateToProps = (state) => ({
   numWifi: countWifi(state),
   numWifiDwellers: countWifiDwellers(state)
 });
-export default connect((mapStateToProps), {
-  eventsRequest,
-  readersRequest,
-  tagsRequest,
-  usersRequest
-})(DashboardView);
+export default connect((mapStateToProps), {})(DashboardView);
```

Comment:
why is `(mapStateToProps)` in brackets? is that a special syntax for something new and shiny in node 6/babel or ?